requirements/dev.txt

```diff
@@ -197,9 +197,9 @@ pep517==0.13.0
     # via
     #   -r requirements/pip-tools.txt
     #   build
-pip-tools==6.9.0
+pip-tools==6.10.0
```

Comment:
No breaking change in the new version: https://github.com/jazzband/pip-tools/blob/main/CHANGELOG.md#6100-2022-11-13